index.html

```diff
@@ -202,12 +218,14 @@ <h4>About me</h4>
 
 <!---------  contact form ------------------------------------->
 <form action="https://formspree.io/f/mqkoavny" method="post">
-    <ul>
-        <li><h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></li>
-        <li><input type="text" name="user-name" id="user-name" placeholder="Full name" required maxlength="30"></li>
-        <li><input type="email" name="email" id="email" placeholder="Email address" required></li>
-        <li><textarea name="message" id="message" maxlength="500" placeholder="Enter text here" required></textarea></li>
-        <li><button id="get-in-touch" name="get-in-touch">Get in touch</button></li>
+    <ul class="contact-ul">
+      <div class="contact-left"> <li><h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></li>
+     </div>
+       <div class="contact-right"> <li><input type="text" name="user-name" id="user-name" placeholder="Full name" required maxlength="30"></li>
+            <li><input type="email" name="email" id="email" placeholder="Email address" required></li>
+            <li><textarea name="message" id="message" maxlength="500" placeholder="Enter text here" required></textarea></li>
+            <li><button id="get-in-touch" name="get-in-touch">Get in touch</button></li>
```

Comment:
- [ ] As suggested by the previous reviewer, please ensure your contact form button is aligned properly.



|  | See below  |
| ------------- | ------------- |
| Your Design | <img width="946" alt="image" src="https://user-images.githubusercontent.com/52934128/214807774-12ad8925-1543-4519-9fd4-28934e5cb901.png">  |
| Figma Design | <img width="517" alt="image" src="https://user-images.githubusercontent.com/52934128/214808272-b6278d96-467a-44a2-bd42-956407ef379b.png">  |